src/components/form/validation.js

```diff
@@ -1,6 +1,6 @@
 const Validation = {
   validName: (value) => {
-    const re = /[\w]+[ ][\w]+/;
+    const re = /^[a-zA-ZàáâäãåąčćęèéêëėįìíîïłńòóôöõøùúûüųūÿýżźñçčšžÀÁÂÄÃÅĄĆČĖĘÈÉÊËÌÍÎÏĮŁŃÒÓÔÖÕØÙÚÛÜŲŪŸÝŻŹÑßÇŒÆČŠŽ∂ð ,.'-]+[ ][a-zA-ZàáâäãåąčćęèéêëėįìíîïłńòóôöõøùúûüųūÿýżźñçčšžÀÁÂÄÃÅĄĆČĖĘÈÉÊËÌÍÎÏĮŁŃÒÓÔÖÕØÙÚÛÜŲŪŸÝŻŹÑßÇŒÆČŠŽ∂ð ,.'-]+$/;
```

Comment:
if we have to use so many special characters in the regexp, why not just simply remove the validation in the name all together? Do we need to validate someone's name?

Reply:
hmmm it's nice to have [First][ ][Last] names validated I guess